jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketConstants.java

```diff
@@ -0,0 +1,21 @@
+/******************************************************************************* 
+ * Copyright (c) 2022 Giancarlo Pernudi Segura and others. 
+ * 
+ * This program and the accompanying materials are made available under the 
+ * terms of the Eclipse Public License v. 2.0 which is available at 
+ * http://www.eclipse.org/legal/epl-2.0. 
+ * 
+ * SPDX-License-Identifier: EPL-2.0 
+ * 
+ * Contributors: 
+ *     Giancarlo Pernudi Segura - initial API and implementation 
+ *******************************************************************************/
```

Comment:
/*******************************************************************************
 * Copyright (c) 2022 IBM Corporation and others.
 *
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License v. 2.0 which is available at
 * http://www.eclipse.org/legal/epl-2.0.
 *
 * SPDX-License-Identifier: EPL-2.0
 *
 * Contributors:
 *     Giancarlo Pernudi Segura
 *******************************************************************************/

Reply:
It should be IBM Corporation now 👍